tests/easier68k/core/models/test_list_file.py

```diff
@@ -184,12 +185,13 @@ def test_list_file_json():
     # this is done because json dumps would not correctly order the sub dictionaries sometimes
     # so instead this just compares the values of the two
     a_val = json.loads(a.to_json())
-    expected_val = json.loads('{"data": {"4096": "010203040506", "4608": "DEADBEEF", "12288": "AAAAAAAAAAAAAAAAAAAAAAAA", "13568": "AAAAAAAAAAAAAAAAAAAAAAAB"}, "symbols": {"DataA": 4096, "DataB": 4608}}')
+    print(a.to_json())

```

Comment:
Remove prints from tests? Or does it not matter as much in tests, only in actual code?

Reply:
I would think having prints in automated tests is a generally bad idea. It would clutter up the console output when tests are run.

Reply:
fwiw pytest captures all of the output, but yeah I was using it to debug tests. will remove it